site/docs/webrtc/webhooks.md

```diff
@@ -0,0 +1,65 @@
+---
+id: webhooks
+title: Webhooks
+slug: /webrtc/webhooks
+description: A brief description of the Webhooks registered by WebRTC API calls.
+keywords:
+  - bandwidth
+  - webrtc
+  - video
+  - webhook
+image: ../../static/img/bandwidth-logo.png
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+## Subscribing to Webhooks
+
+The only API call that allows the registration of a callback / webhook is the POST to the participants resource that is used to create a new participant. The callback/webhook URL is declared in the `POST` to the `https://api.webrtc.bandwidth.com/v1/accounts/{accountId}/participants` endpoint. If the `callbackUrl` property is included in the payload, the value associated with that property is assumed to be a URL. That URL (if found to be valid) will be invoked on events that occur as part of the Participant life cycle.
```

Comment:
I would just refer to it as createParticipant endpoint, as it is labeled in the API Docs

Reply:
done